crypto/types/src/lib.rs

```diff
@@ -30,5 +40,5 @@ pub mod prelude {
     pub use super::utils::*;
     pub use super::vrf::*;
 
-    //pub use libp2p_identity::PeerId;
+    pub use libp2p_identity::PeerId;
```

Comment:
This is an unpleasant dependency in the crypto lib.

Reply:
I can remove it, but the conversion from `PeerId` to `OffchainPublicKey` is non-trivial.